flow-report/src/summary/summary.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {FunctionComponent} from 'preact';
+import {Gauge} from '../lh-wrappers';
+import {FlowStepIcon} from '../sidebar/flow';
+import {useDerivedStepNames, useFlowResult} from '../util';
+import {Util} from '../../../report/renderer/util';
+
+const DISPLAYED_CATEGORIES = ['performance', 'accessibility', 'best-practices', 'seo'];
+
+const SummaryFlowIcon: FunctionComponent<{
+  hideTopLine: boolean,
+  hideBottomLine: boolean,
+  mode?: LH.Result.GatherMode
+}> = ({mode, hideBottomLine, hideTopLine}) => {
+  return (
+    <div className="SummaryFlowIcon">
+      <div
+        className="SummaryFlowIcon__line"
+        style={hideTopLine ? {background: 'transparent'} : undefined}
+      />
+      {
+        mode && <FlowStepIcon mode={mode}/>
+      }
+      <div
+        className="SummaryFlowIcon__line"
+        style={hideBottomLine ? {background: 'transparent'} : undefined}
```

Comment:
could this instead be done with CSS :first-child and :last-child ?

Reply:
Yes, will probably update `SidebarFlow` with this as well.